### PR TITLE
Endpoint playground

### PR DESCRIPTION
**Pull request by zhendi**
Add endpoint playground component

image

### CODE REVIEW

File: app/javascript/components/endpoints/playground/ApiExample.vue [hunk 1]
```diff
@@ -0,0 +1,233 @@
+<template>
+  <div
+    class="h-[77px] w-full p-4 bg-white border-b border-[#eaecf0] items-center gap-2 inline-flex rounded-tr-2xl"
+  >
+    <SvgIcon name="playground_api" />
+    <div class="text-[#344053] text-base font-medium leading-normal">
+      Send requests to endpoint
```
Comment (ymh6315431) [inline app/javascript/components/endpoints/playground/ApiExample.vue:7]: i18n

File: app/javascript/components/endpoints/playground/TestEndpoint.vue [hunk 2]
```diff
@@ -0,0 +1,208 @@
+<template>
+  <div
+    class="h-[77px] w-full p-4 bg-white border-b border-[#eaecf0] items-center gap-2 inline-flex rounded-tr-2xl"
+  >
+    <SvgIcon name="playground_test" />
+    <div class="text-[#344053] text-base font-medium leading-normal">
+      Test your endpoint
```
Comment (ymh6315431) [inline app/javascript/components/endpoints/playground/TestEndpoint.vue:7]: i18n

File: app/javascript/components/endpoints/playground/ApiExample.vue (hunk 1 above)
Comment (zhendi) [reply app/javascript/components/endpoints/playground/ApiExample.vue]: 已更新

**Review by ymh6315431 — APPROVED** [review]
LGTM

**Pull request merged**